src/drawer.js

```diff
@@ -97,6 +97,13 @@ $.Drawer = function( options ) {
     this.context    = this.useCanvas ? this.canvas.getContext( "2d" ) : null;
 
     /**
+     * Sketch canvas used to temporarily draw tiles which cannot be drawn directly
+     * to the main canvas due to opacity.
+     */
+    this.sketchCanvas = this.useCanvas ? document.createElement( "canvas" ) : null;
```

Comment:
Since adding a new Canvas can have a big impact on our memory footprint, I think it would be good if we could do it only as needed, i.e. the first time we have a fractional opacity.